docs/usage/editing-paths.md

```diff
@@ -0,0 +1,78 @@
+Choreo uses trajectory optimization find the optimal route through waypoints. Additionally, Choreo allows fine-grained customization while still maintaining concrete stability. This allows for smoother paths and better odometry.
+
+## Waypoints
+
+Waypoints are an ordered position that you'd like to hit on your path. There are multiple types of waypoints based on what you'd like to constrain in your path.
+
+To add a waypoint, select the type of waypoint from the waypoints navbar on the top to get started, then click on the field where you want it. Your waypoints will show up on the sidebar. A yellow dot represents the currently selected waypoint, green for the starting waypoint, and red for the ending waypoint.
+
+![waypoints navbar](../media/waypoints+navbar.png)
+
+### Pose Waypoints
+
+![pose waypoint](../media/pose_waypoint.png)
+
+Pose Waypoint is the first button on the waypoints navbar and consists of a translation and a rotation. Drag the little yellow circle (outlined in black) to change the heading.
+
+- **X and Y**: position in meters from the bottom left (origin)
+- **θ**: Angular offset in radians -pi to pi, 0 to right
+
+### Translation Waypoints
+
+![translation waypoint](../media/translation_waypoint.png)
+
+Translation Waypoint is the second button on the waypoints navbar and consists of a translation but not a rotation. The UI still includes the yellow circle, but isn't only stored to be able to convert later into a Pose Waypoint. However, rotation is still used as part of the initial guess.
```

Comment:
Do you mean "it is" rather than "isn't" in the second sentence? As is this doesn't make sense.